src/travel-details-map-screen/TravelDetailsMapScreenComponent.tsx

```diff
@@ -90,14 +91,38 @@ export const TravelDetailsMapScreenComponent = ({
   });
 
   const [shouldTrack, setShouldTrack] = useState<boolean>(true);
+  const [zoomLevel, setZoomLevel] = useState<number>(FOLLOW_ZOOM_LEVEL);
   const [cameraState, setCameraState] = useState<{
```

Comment:
This could be renamed as [cameraHeading, setCameraHeading] for a proper name

Reply:
Good point, updated now